config-files/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+### PR Checklist
+
+- [ ] JIRA Ticket(s) included in PR Title and are formatted correctly if applicable (JIRA-123)
```

Comment:
Well I meant the PR submitter can remove the line entirely so it's not just checked "because"
...
or they can just check it because